tests/tnimterop_c.nim

```diff
@@ -137,7 +138,7 @@ var
   k: uKernel
   kp: Kernel
 
-cAddStdDir()
+static: cAddStdDir()
 
```

Comment:
Can be removed since it is being tested in tmath.

Reply:
done, good idea